changelog/2.006.dd

```diff
@@ -31,8 +31,7 @@ $(WHATSNEW
 	$(LI Added the
 	$(LINK2 $(ROOT_DIR)phobos/std_variant.html, std.variant) module.)
 	$(LI Incorporated many of the Tango GC structural differences (much more to go still).)
-	$(LI Added the
-	$(LINK2 $(ROOT_DIR)phobos/std_contracts.html, std.contracts) module.)
+	$(LI Added the $(D std.contracts) module.)
```

Comment:
So, AFAIU, all links to now non-existing modules, should be changed to plain code font.